metadata.json

```diff
@@ -8,9 +8,7 @@
     "project_page": "https://github.com/chriskuehl/puppet-spiped",
     "issues_url": "https://github.com/chriskuehl/puppet-spiped/issues",
     "dependencies": [
-        {
-            "name": "puppetlabs/stdlib",
-            "version_requirement": ">= 2.3.1"
-        }
+        { "name": "puppetlabs/stdlib", "version_requirement": ">= 2.3.1 < 7.0.0" },
+        { "name": "camptocamp/systemd", "version_requirement": ">= 1.1.0 < 3.0.0" }
```

Comment:
for a new dependency, I always pin to the newest version because I know for 100% that this one is working in this module.

Reply:
I bumped the minimum version to `2.1.0`, (not the very latest, but a fair bit newer than `1.1.0`), and amended my commit.  Unfortunately, I completely forgot to `git add` the change and only noticed after I had merged the PR!  I'll slip this minimum version bump into a different PR before our first release.